gravitee-apim-console-webui/src/management/api/subscriptions/edit/api-subscription-edit.component.ts

```diff
@@ -228,9 +228,14 @@ export class ApiSubscriptionEditComponent implements OnInit {
         takeUntil(this.unsubscribe$),
       )
       .subscribe(
-        (_) => {
-          this.snackBarService.success(`Subscription validated`);
-          this.ngOnInit();
+        (subscription) => {
+          if (subscription.status === 'ACCEPTED') {
+            this.snackBarService.success(`Subscription validated`);
+            this.ngOnInit();
+          } else {
+            this.snackBarService.error('Subscription ' + subscription.status);
+            this.ngOnInit();
```

Comment:
When we cancel the popin, we should not have subscription (see `EMPTY` line 226) so I guess we should handle that case 🤔 
Also my we can extract `this.ngOnInit()` outside if/else to prevent duplication 🤷

Reply:
Yeah, sure I missed that 🙈

About the `EMPTY` I'm not sure exactly if we need to handle it anyway. When I tested closing the popup locally it seemed working. From what I read `EMPTY` emits the complete so we can technically handle it somehow as I did in line 240 but I'm not sure if this is necessary. Maybe return false, wdyt? 
 
<img width="772" alt="image" src="https://github.com/user-attachments/assets/5b6f80c4-8221-4412-8c60-5dc4090d8a95">

Reply:
If we have a test case handling that case, I'm OK to merge like this. I didn't check the existing test cases to confirm it is covered or not.

Reply:
I wasn't sure, but looks like we're covered 😉
<img width="1054" alt="image" src="https://github.com/user-attachments/assets/8356df54-62db-4e38-a1e1-0d6b1d708f3b">